dockerbuild/ssp-overrides/config.php

```diff
@@ -70,6 +70,7 @@
 $GOOGLE_ENABLE = Env::get('GOOGLE_ENABLE', false);
 $HUB_MODE = Env::get('HUB_MODE', false);
 $ANALYTICS_ID = Env::get('ANALYTICS_ID', null);
+$ANALYTICS_ID_GA4 = Env::get('$ANALYTICS_ID_GA4', null);
```

Comment:
I think the default value for `Env::get()` is already `null`, so you can remove the 2nd parameter if you want to. I don't know why it was included on the call above as well.